spec/lib/msf/base/sessions/command_shell_spec.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+COMMANDS = %i[help background sessions resource shell download upload source irb pry]
```

Comment:
Generally we avoid introducing top level globals as it can impact the implementation you're testing 👀 

Might be cleaner to move this into the `RSpec.describe` block or swap to using it_behaves_like